### ISSUE TITLE
Tweak Wind Turbines

### ISSUE THREAD

**Issue by jrddunbr** [age-series/ElectricalAge#41]
Wind Turbines are poorly balanced.

Double their physical size, make them 200v or 800v, and generate *at least* 1-2kW. IRL, they can generate 2-3MW for the bigger ones. 50W is stupid.

Should make it really important that the space in front of them is clear. Expand the requirement from 5 blocks to at least 16 blocks.

**Comment by Grissess** [comment]
I like the idea of making them 800V, which makes them more practically accessible at a higher tier, and thus this increase in power input would be a reward for investing enough in infrastructure to have 800V generation. I'm not entirely sure, but I suspect it would be one of the few direct-to-800V generators that exist in the game, if that is done.

"Wind power" likely modulates a resistor on the output terminal to a voltage source. Because of drop, 800V is nominal, of course.

**Comment by OmegaHaxors** [comment]
They're way way way stronger than the stats imply, and are balanced around realism. Put them up higher and wait for a rain storm and they can destroy the output of the solar panels.

I wouldn't mind having a larger one with a higher voltage, but the 50V one is actually insanely good.

**Comment by jrddunbr** [comment]
50w sounds like something you would get from a desk fan. I was suggesting a more industrial size like the ones near where I live.

**Comment by OmegaHaxors** [comment]
Fine but leave the 50v ones alone, they are very good for the early game.

**Comment by jrddunbr** [comment]
Yeah sure

**Comment by jrddunbr** [comment]
@OmegaHaxors says he wants to take this one over a few days back.

**Comment by OmegaHaxors** [comment]
Want to and capable to are not quite the same in this situation. I've got a ton of other stuff to work on right now sadly, but when things slow down for me i'll be able to get some work done on it.

TL;DR don't wait on me

**Comment by jrddunbr** [comment]
This issue is being closed now since there will be no more feature updates in Electrical Age for Minecraft 1.7.10.
If the issue is relevant to 1.12, it will be re-opened there at a later date.

**Comment by jrddunbr** [comment]
Addressed in 456e0d5
